sdk/src/client/bindings/java/lib/src/main/java/org/iota/types/addresses/AliasAddress.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package org.iota.types.addresses;
+
+import org.iota.types.addresses.Address;
+import org.iota.types.ids.AliasId;
+
+public class AliasAddress extends Address {
+
+    private int type = 8;
+    private AliasId aliasId;
+
+    public AliasAddress(AliasId aliasId) {
+        this.aliasId = aliasId;
+    }
+
+    public int getType() {
+        return type;
```

Comment:
should move getType to Address itself

Reply:
And how assign the correct type then for each type?
Just adding it to Address doesn't work `java.lang.IllegalArgumentException: class org.iota.types.addresses.Ed25519Address declares multiple JSON fields named type`